src/index.js

```diff
@@ -1,17 +1,53 @@
 import './style.css';
+import { MyHttpRequest } from './modules/httpRequests.js';
+import { DomRequest } from './modules/domRequests.js';
+import { Templates } from './modules/domTemplates.js';
 
-// Form Validators
+const myUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/dJgNxV0NZwLdKmSLhQCZ/scores/';
+const httpreq = new MyHttpRequest(myUrl);
+const myForm = document.querySelector('#myForm');
+
+window.refreshData = () => {
+  DomRequest.clear('tableElements');
+  httpreq.getAsync().then((res) => {
+    res.forEach((element) => {
+      DomRequest.appendTemplate('tableElements', Templates.scoreRow(element.user, element.score));
+    });
+  });
+};
+
+window.onload = () => {
+  window.refreshData();
+};
+
+myForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (!myForm.checkValidity()) {
+    return;
+  }
+  const userName = document.querySelector('#TextBoxName');
+  const userScore = document.querySelector('#TextBoxScore');
+  httpreq.postAsync({
+    user: userName.value,
+    score: parseInt(userScore.value, 10),
+  }).then(() => {
+    DomRequest.appendTemplate('tableElements', Templates.scoreRow(userName.value, userScore.value));
+    myForm.reset();
+  });
+});
+
+// #region Form-Validators
 (function () {
```

Comment:
- [ ] Please use arrow funtions in all your funtion declarations, this is part of this module to help you learn ES6 syntax 🤓 
    - Kindly take a look to your JS files and change all the functions to ES6 syntax 😄